src/dataprep/temppipe.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
The file is a "helper" when we run single scripts, but not relevant otherwise. I suggest either we delete or rename it and document its purpose at the beginning of the file